projects/angular/src/layout/breadcrumbs/breadcrumb-item.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'clr-breadcrumb-item',
+  template: `
+    <ng-content />
+  `,
```

Comment:
```suggestion
  template: '<ng-content />',
```

This is can be one line.